manifests/istio-policy/templates/deployment.yaml

```diff
@@ -218,9 +218,11 @@ spec:
         - name: sds-uds-path
           mountPath: /var/run/sds
           readOnly: true
+        {{- if eq .Values.global.jwtPolicy "third-party-jwt" }}
```

Comment:
Please also set condition for:
https://github.com/istio/istio/blob/19cc5c9f7f0d903f159175fb259f3e5982f91f3b/manifests/istio-policy/templates/deployment.yaml#L50

Reply:
Done.